### PR TITLE
Fix SuperSimpleNet pretrained weights.

### LINKED ISSUE #2688: [Bug]: Unable to reproduce the results of supersimplenet on the MVTec dataset.
### Describe the bug

I’m trying to reproduce the reported results on the MVTec dataset using the `supersimplenet` model, but unfortunately, I haven’t been able to achieve similar performance. During training, it appears that the loss does not converge, even after several epochs.

Could you please share more details about the training or evaluation setup? 

### Dataset

MVTecAD

### Model

Other (please specify in the field below)

### Steps to reproduce the behavior

none

### OS information

OS information:
- OS: [Ubuntu 20.04]
- Python version: [3.10.0]
- Anomalib version: [2.1.0.dev0]
- PyTorch version: [2.5.0]
- CUDA/cuDNN version: [12.4]
- GPU models and configuration: [Tesla V100]

### Expected behavior

none

### Screenshots

_No response_

### Pip/GitHub

GitHub

### What version/branch did you use?

2.1.0.dev0

### Configuration YAML

```yaml
none
```

### Logs

```shell
none
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by blaz-r**
## 📝 Description

- This PR adds a fix for SuperSimpleNet, where incorrect pretrained weights were used.
- 🛠️ Fixes #2688 

## ✨ Changes

Select what type of change your PR is:

- [x] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔨 Refactor (non-breaking change which refactors the code base)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 Documentation update
- [ ] 🔒 Security update

## ✅ Checklist

Before you submit your pull request, please make sure you have completed the following steps:

- [ ] 📋 I have summarized my changes in the [CHANGELOG](https://github.com/open-edge-platform/anomalib/blob/main/CHANGELOG.md) and followed the guidelines for my type of change (skip for minor changes, documentation updates, and test enhancements).
- [ ] 📚 I have made the necessary updates to the documentation (if applicable).
- [ ] 🧪 I have written tests that support my changes and prove that my fix is effective or my feature works (if applicable).

For more information about code review checklists, see the [Code Review Checklist](https://github.com/open-edge-platform/anomalib/blob/main/docs/source/markdown/guides/developer/code_review_checklist.md).

### CODE REVIEW

File: src/anomalib/models/image/supersimplenet/lightning_model.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ class Supersimplenet(AnomalibModule):
 
     Args:
         perlin_threshold (float): threshold value for Perlin noise thresholding during anomaly generation.
-        backbone (str): backbone name
+        backbone (str): backbone name. IMPORTANT! use only backbones with torchvision V1 weights ending on ".tv".
```
[the remark this answers is not in the document]
Comment (blaz-r) [reply src/anomalib/models/image/supersimplenet/lightning_model.py:64]: The problem with V2 weights is that their feature values are, for some reason, an order of magnitude larger than V1. In some preliminary testing, I found out that you can just multiply the noise std by 10, to correct for this shift. However, we've not fully tested with V2 weights on all data, so I'm not sure if it will always work. Therefore, I think it's best to stick to V1.

[the remark this answers is not in the document]
Comment (samet-akcay) [reply src/anomalib/models/image/supersimplenet/lightning_model.py:64]: ok thanks for the clarification

**Review by samet-akcay — APPROVED** [review]
thanks for addressing this

**Review by samet-akcay — APPROVED** [review]
